template/comment.create/body.html

```diff
@@ -1,134 +1,148 @@
-                  color: purple;
-                  text-decoration: none;">
-              {{lower .spaceOwner.Data.Attributes.Username}}{{areaPath .workitemArea.Data.Attributes.ParentPathResolved .workitemArea.Data.Attributes.Name}}
-            </a>
+          <span>
+              <a href="{{.webURL}}/{{.spaceOwner.Data.Attributes.Username}}/{{.space.Data.Attributes.Name}}/plan?area={{.workitemArea.Data.Attributes.Name}}" class="link-area">
+                {{lower .spaceOwner.Data.Attributes.Username}}{{areaPath .workitemArea.Data.Attributes.ParentPathResolved .workitemArea.Data.Attributes.Name}}
+              </a>
+            </span>
           </span>
-        </span>
-      </p>
-    </div>
-    <div style="
-          background-color: #f9f9f9;
-          padding: 10px 20px;">
-      <h3 style="margin-bottom: 5px;">
-        <a href="{{detailURL .webURL .spaceOwner .space .workitem}}" style="
-            color: #0088ce;
-            font-weight: 700;
-            text-decoration: none;">
-          {{index .workitem.Data.Attributes "system.title"}}
+        </p>
+      </div>
+      <div class="content">
+        <h3>
+          <a href="{{detailURL .webURL .spaceOwner .space .workitem}}" class="link">
+            {{index .workitem.Data.Attributes "system.title"}}
+          </a>
+        </h3>
+        <span style="font-size: 12px; color: #8b8d8f;">comment created at {{date .comment.Data.Attributes.CreatedAt}}</span>
+        <p style="font-size: 16px;">
+          {{raw .comment.Data.Attributes.BodyRendered}}
+        </p>
+      </div>
+      <div class="footer">
+        <a href="{{detailURL .webURL .spaceOwner .space .workitem}}" class="link-footer">
+          Link to this item
         </a>
-      </h3>
-      <span style="font-size: 12px; color: #8b8d8f;">comment created at {{date .comment.Data.Attributes.CreatedAt}}</span>
-      <p style="
-          font-size: 16px;
-          text-align: left;
-          min-height: 150px;
-          max-width: 700px;">
-        {{raw .comment.Data.Attributes.BodyRendered}}
-      </p>
-    </div>
-    <div style="
-            margin-top: 25px;
-            margin-bottom: 5px;
-            display: inline-flex;">
-      <span style="width: 150px;">
-        <a href="{{detailURL .webURL .spaceOwner .space .workitem}}"
-            style="
-              color: #0088ce;
-              text-decoration: none;
-              font-weight: 500;
-              vertical-align: sub;">
-              Link to this item</a>
-      </span>
-      <span style="width: 150px;">
-        <a href="{{detailURL .webURL .spaceOwner .space .workitem}}"
-            style="
-              color: #0088ce;
-              text-decoration: none;
-              font-weight: 500;
-              vertical-align: sub;">
-              Comment</a>
-      </span>
+        <span class="divider-vertical"></span>
+        <a href="{{detailURL .webURL .spaceOwner .space .workitem}}" class="link-footer">
```

Comment:
why two different paths to same place? `Link to this item` and `Comment` are same links,

Reply:
In the future, the idea is that clicking on 'Comment' will bring the user to the work item and automatically activate a new comment field.